### ISSUE TITLE
Dynamic lights stays after exiting world

### ISSUE THREAD

**Issue by P0stf1x** [LambdAurora/LambDynamicLights#30]
When I exit the world with anything sourcing dynamic light, the light stays and nothing removes it.
edit: nothing but restart
 ![Screenshot](https://user-images.githubusercontent.com/46126263/96667321-8a6bad00-1361-11eb-87b7-1cb7c9498b8e.png)

**Comment by LambdAurora** [comment]
That's... concerning, it should not happen as this mod doesn't store anything in the world, and doesn't affect the actual world lighting (as the client & server block light is always 0).

I'm not sure of what is happening there, a complete list of mods (with their version) might be useful.

**Comment by P0stf1x** [comment]
I removed every mod but dynamic lights and it still happens. Probably something with my system or java? Reinstalled windows since last time played, so I can't say exactly

Mods:<details>
![изображение](https://user-images.githubusercontent.com/46126263/96942747-aa74ab00-14de-11eb-9b50-27b8e2ea8991.png)
</details>

System info:
<details>

```
Processor	AMD FX(tm)-6300 Six-Core Processor 4.20 GHz
Installed RAM	8,00 GB
System type	64-bit operating system, x64-based processor
Edition		Windows 10 Pro
Version		20H2
OS build	19042.572
```

</details>
Java info:
<details>

```
java version "1.8.0_261"
Java(TM) SE Runtime Environment (build 1.8.0_261-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.261-b12, mixed mode)
```

</details>

**Comment by LambdAurora** [comment]
Does the light stay even without LambDynamicLights?
Does it stays across restarts?

**Comment by P0stf1x** [comment]
It stays until I restart Minecraft. Reloading resource packs (F3+T), reloading chunks (F3+A), changing render distance doesn't affect it. Changing dynamic lights quality (fastest/fast/fancy) does nothing, but if I set it to off, the light disappears. Even if just scrolled through off option

![screenshot](https://user-images.githubusercontent.com/46126263/96994632-a9bb3380-1535-11eb-91e5-99748489ea5f.png)

**Comment by LambdAurora** [comment]
Might be an issue with the dynamic light sources cache not getting cleared up correctly, which is highly concerning and in theory should crash. The OFF option will entirely clear the cache.

**Comment by P0stf1x** [comment]
Now it became even stranger. I set around 10 light bugs and switch the mod off, but it wouldn't disappear until I place a torch

<details>

![изображение](https://user-images.githubusercontent.com/46126263/96996379-7037f780-1538-11eb-8ee9-aa164bc656f0.png)
![изображение](https://user-images.githubusercontent.com/46126263/96996401-77f79c00-1538-11eb-9e89-f91c86822710.png)
![изображение](https://user-images.githubusercontent.com/46126263/96996469-95c50100-1538-11eb-9e5d-ddde2e516c38.png)

</details>

**Comment by LambdAurora** [comment]
Good news! It's now fixed in release [1.3.2](https://github.com/LambdAurora/LambDynamicLights/releases/tag/1.3.2)!
